server/routes/routerRequestHandlers/retrieveEducation.ts

```diff
@@ -9,11 +9,12 @@ import asyncMiddleware from '../../middleware/asyncMiddleware'
 const retrieveEducation = (educationAndWorkPlanService: EducationAndWorkPlanService): RequestHandler => {
   return asyncMiddleware(async (req: Request, res: Response, next: NextFunction) => {
     const { prisonNumber } = req.params
+    const { username } = req.params
```

Comment:
Almost right 😁 , except the username wont come from `req.params` (`req.params` are the path parameters on the URL - eg: `https://learning-and-work-progress-dev.hmpps.service.justice.gov.uk/prisoners/G2385GG/create-induction/affect-ability-to-work`, you can see the prison number (`G2385GG`) in the path. 
We don't have the username in any of our URL paths.

The user object is on the request, and it in turn has a username property - `req.user.username`
Have a look at what you did in your last PR in `TimelineController.ts` for an example:
```
    const timeline = await this.timelineService.getTimeline(prisonNumber, req.user.username)
```

Your test above will also need to change based on this